cgroup.go

```diff
@@ -247,6 +256,7 @@ func (c *cgroup) Delete() error {
 			if err := remove(path); err != nil {
 				errs = append(errs, path)
 			}
+			continue
```

Comment:
is this `continue` necessary? I don't see how it changes loop flow since this is the end of the `for` and will naturally continue through the end of the range of subsystems.

Reply:
It's not strictly necessary, no. I added it to because from my interpretation of the structure of the code here, it's sort of like a switch case and only one branch should be valid, so if there was ever a third case (very unlikely since cgroup v1 is now legacy code), this would be correct for that situation.

Looking at it a bit more, I think it would be better if those two `if` blocks were restructured to better represent that intent that only one interface is expected to be matched. What do you think of something more like this, would that be more readable?
```go
switch s.(type) {
case deleter:
    ...
case pather:
   ...
}
```

Reply:
Checked @estesp offline and this is not a blocker.